### PR TITLE
develop docs: update Cloud UI

### PR DESCRIPTION
**Pull request by Oreoxmt**
Signed-off-by: Aolin <aolin.zhang@pingcap.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
-3. 在 [Clusters](https://tidbcloud.com/console/clusters) 页面中点击 **Create Cluster** 按钮。
+3. 在 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中点击 **Create Cluster** 按钮。
 4. **Create Cluster** 页面默认选择 **Serverless Tier**。你可以根据需要修改集群名称并选择可用区。
 5. 点击 **Create** 创建 Serverless Tier 集群。
 
     你的 TiDB Cloud 集群将于 30 秒后创建完毕。
 
-6. 集群创建完毕后，点击 **Security Settings** 按钮，在对话框中设置连接集群的 root 密码，完成后点击 **Submit**。如果没有设置 root 密码，你将无法连接集群。
-7. 点击 **Connect** 按钮。在连接对话框 **Connect with a SQL Client** 下选择你需要的连接方式并保存对应的连接字符串。下面连接到集群以 MySQL 客户端为例。
+6. 集群创建完毕后，点击 **...** 按钮，在下拉框中选择 **Security Settings** 按钮。在对话框中设置连接集群的 root 密码，完成后点击 **Apply**。如果没有设置 root 密码，你将无法连接集群。
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:25]:
```suggestion
6. 集群创建完毕后，点击右上角的 **...** 按钮，在下拉框中选择 **Security Settings**。在对话框中设置连接集群的 root 密码，完成后点击 **Apply**。如果没有设置 root 密码，你将无法连接集群。
```

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
...
+7. 点击 **Connect** 按钮，在连接对话框中选择你需要的连接方式和操作系统并保存对应的连接字符串。下面连接到集群步骤以 MySQL 客户端为例。
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:26]:
```suggestion
7. 点击 **Connect** 按钮，在连接对话框中选择你需要的连接方式和操作系统并保存对应的连接字符串。下面连接到集群的步骤将以 MySQL 客户端为例。
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**